src/main/resources/org/jenkinsci/plugins/pitmutation/targets/MutationResult/index.jelly

```diff
@@ -64,7 +69,7 @@
                         <td>
                             <j:if test="${child.statsDelta.undetected &gt; 0}">+</j:if>${child.statsDelta.undetected}
                         </td>
-                        <td data="${child.mutationStats.killPercent}">
+                        <td style="color:#${child.mutationStats.killPercentTextColor}; background-color:#${child.mutationStats.killPercentFillColor};" data="${child.mutationStats.killPercent}">
```

Comment:
Typically one would solve this using an external css file and use a specific class. But since the colors are already provided in Java it is ok.